src/Pages/AboutPage/about.scss

```diff
@@ -0,0 +1,38 @@
+.about_contact__map, .about_contact__social__address, .about_contact__social__phone, .about_contact__social__media {
+    display: flex;
+}
+
+.about {
+    max-width: 100%;
+    display: flex;
+    justify-content: space-around;
+    margin:20px 135px 40px 135px;
+
+    .about_text {
+        max-width: 579px;
+        padding-right: 140px;
+
+        .about_text__text-section {
+            margin-top: 20px;
+        }
+    }
+}
+
+.about_contact__social__address {
+    margin-top: 29px;     
+}
+
+.about_contact__social__phone{
+    margin-top: 16px;
+}
+
+.about_contact__social__media {
+    margin-top: 32px;
+}
+
+.about-icons {
```

Comment:
make it `.about-icon` to make it obvious that it's not a container for icons but icon itself